values/src/context.rs

```diff
@@ -32,7 +32,13 @@ pub struct ResolverContext {
     ctx: Context<Local>,
 }
 
-impl ResolverContext {
+impl ResolverContext { 
+
+    pub fn from_nsecs(nsecs: i64) -> ResolverContext {
```

Comment:
I got confused reading this as I didn't understand why it was nano seconds, and then I realized `nsecs` <=> `number of seconds`, so I think the naming is misleading.
What about `from_timestamp_in_seconds(timestamp: i64)` ?